app/assets/stylesheets/common/admin/admin_table.scss

```diff
@@ -97,6 +97,7 @@
   }
 
   // Success badge
+
   .status-label.--success {
```

Comment:
Separating the comment from the block it describes is not ideal...

Reply:
yes agreed will see what I can do about this